backend.py

```diff
@@ -271,9 +271,11 @@ def get_tiles(self, coordinates):
 
     def get_active_robots(self):
         """
-        Return a list of active robots.
+        Create generator function for active robots.
```

Comment:
Už jsem měl čas popřemýšlet jak stručně/jednoznačně popsat generátorové funkce, tak navrhuju:
```suggestion
        Yield all active robots.
```

Reply:
Tak to mi vysvětlete co jaký generátor a co ten yield, sice jsem to jednou viděla ale proč co  😁

Reply:
Je to na začátku https://www.youtube.com/watch?v=0EdYPukCQHg
Kde přesně se začínáš ztrácet?

Reply:
No nechápu k čemu je to dobré prakticky přímo v našem případě než když se vrátí rovnou seznam robotů... jo ahá, když to tak píšu... my to potřebujeme kontrolovat pořád jestli jsou aktivní, tak už asi dobré :)